protocol/avdl/chat1/common.avdl

```diff
@@ -593,7 +600,10 @@
 
   record ChatSearchInboxHit {
     ConversationID convID;
+    TeamType teamType;
     string convName;
+    string query;
+    gregor1.Time time;
```

Comment:
`ctime`?

Reply:
It's not really a ctime, its just the time used in the sort. We use the same name on regular inbox items.